custom/aaa/tests/data/agg_tables/womanhistory.csv

```diff
@@ -0,0 +1,2 @@
+person_case_id,fp_current_method_history,fp_preferred_method_history,family_planning_form_history
+ec_using_condoms,"[[u'2019-02-03 00:00:00', u'condom']]","[[u'2019-02-03 00:00:00', None]]","[datetime.date(2019, 2, 3)]"
```

Comment:
The unicode ```u``` prefixing is leading to errors in python 3:
```
FAIL: custom.aaa.tests.test_aggregations:AggregationScriptTestBase.test_agg_woman_history_table

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/mnt/commcare-hq-ro/custom/aaa/tests/test_aggregations.py", line 131, in test_agg_woman_history_table

    sort_key=['person_case_id']

  File "/mnt/commcare-hq-ro/custom/aaa/tests/test_aggregations.py", line 117, in _load_and_compare_data

    self._load_csv(path)

  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/__init__.py", line 280, in _fasterAssertListEqual

    self.fail('\n'.join(messages))

AssertionError: 

                    Actual and expected row 1 are not the same

                    Actual:   fp_preferred_method_history: [['2019-02-03 00:00:00', None]], fp_current_method_history: [['2019-02-03 00:00:00', 'condom']], person_case_id: ec_using_condoms

                    Expected: fp_preferred_method_history: [[u'2019-02-03 00:00:00', None]], fp_current_method_history: [[u'2019-02-03 00:00:00', u'condom']], person_case_id: ec_using_condoms
```

@emord Thoughts if there's a better way to do this?

Reply:
Yeah. i'll try to think of a better way to compare. Let me know if it becomes blocking